gulpfile.js

```diff
@@ -135,6 +140,12 @@ gulp.task('build:chrome', ['compress:chrome']);
 
 gulp.task('build:firefox', ['compress:firefox']);
 
+gulp.task('build:extensions', [], function(callback) {
```

Comment:
`gulp.task` without dependencies can drop the `[]`, but we probably want to lint first, no?

So it should be more like `gulp.task('build:extensions', ['lint:scripts'], function(cb) {`